app/scripts/app.js

```diff
@@ -82,6 +82,12 @@
         controller: 'ProjectsController',
         controllerAs: 'project'
       })
+      .when('/events/new', {
+        templateUrl: 'views/events/new.html',
+        controller: 'EventsController',
+        controllerAs: 'vm'
```

Comment:
This name should be related to controller name. It is not related to `vm` variable which defined inside controller.